doc/manual/manual.tex

```diff
@@ -10036,6 +10036,49 @@ \subsection{Extending the basic solver}
 \cite{BHK07,BK99m}.}
 
 
+\subsection{Testing \aspect}
+\label{sec:testing}
+
+\aspect{} makes use of a large suite of tests to ensure correct behavior.
+
+\subsubsection{Running tests}
+\label{sec:running_tests}
+
+In order to run the tests, it is necessary to have either Diff or Numdiff to
+compare the results to the known good case. While it is possible to use Diff,
+Numdiff is preferred due to being able to more accurately identify whether a
+variation in numerical output is significant.
+
+To run a set of tests, go to the build directory and execute 
+
+\begin{verbatim}
+    ctest -R
+\end{verbatim}
+
+This will run all of the existing tests. To only part of the test suite, it is
+possible to supply a regex that will select the names of the desired tests as
+follows
+
+\begin{verbatim}
+    ctest -R <regex>
+\end{verbatim}
+
+
+\subsubsection{Writing tests}
+\label{sec:writing_tests}
+
+% Copied from email response by gassmoeller, and adjusted
```

Comment:
No need for the reference :smile: I do not feel like I made a authorship worthy contribution in that mail.